Hymns/Search/SearchView.swift

```diff
@@ -8,7 +8,7 @@ struct SearchView: View {
     var body: some View {
         NavigationView {
             VStack {
-                SearchBar(text: $viewModel.searchInput)
+                SearchBar(text: $viewModel.searchInput, isFirstResponder: true)
```

Comment:
Do you think we can make this a more informative name? like "showKeyboard" or something, since that's essentially what we're doing. isFirstResponder is a little bit vague

Reply:
other than that lgtm!!!!